README.md

```diff
@@ -1,5 +1,30 @@
 # Analytic Hierarchy Process Solver in Python
 [![license](https://img.shields.io/github/license/mashape/apistatus.svg)]()
 
+## Theory
+Saaty's AHP model for making decision
+[AHP](https://en.wikipedia.org/wiki/Analytic_hierarchy_process)
+
+## Example
+
+### Scripts
```

Comment:
This sub-heading should be more descriptive and include the python import commands. It would be ideal to write about the installation process , i.e. the magic `pip` command before citing these examples.

Reply:
ok